modules/event-settings-wheel-duration-picker/src/TimePickerModule.tsx

```diff
@@ -0,0 +1,33 @@
+import { requireNativeViewManager } from "expo-modules-core"
+import React from "react"
+import { StyleProp, ViewProps, ViewStyle } from "react-native"
+import { NativeViewGestureHandler } from "react-native-gesture-handler"
+
+type OnDurationChangeEvent = {
+  duration: number
+}
+
+export type TimePickerProps = {
+  initialDuration: number
+  onDurationChange: (event: { nativeEvent: OnDurationChangeEvent }) => void
```

Comment:
Do we need to expose the underlying `nativeEvent` here? I think it would be better to wrap this up, and just use the duration value directly as the function parameter.